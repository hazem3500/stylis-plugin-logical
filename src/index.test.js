import { pluginWithPropertyContext } from './index';

describe('padding logical properties to be transformed', () => {
    test("padding-inline-end to be transformed to fallback", () => {
        expect(
            pluginWithPropertyContext(
                'padding-inline-end: 50px;'
            )
        ).toMatchSnapshot();
    });
    test("padding-inline-start to be transformed to fallback", () => {
        expect(
            pluginWithPropertyContext(
                'padding-inline-start: 50px;'
            )
        ).toMatchSnapshot();
    });
})