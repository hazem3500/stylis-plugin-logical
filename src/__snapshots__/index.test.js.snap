// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`padding logical properties to be transformed padding-inline-end to be transformed to fallback 1`] = `
"
        padding-inline-end: 50px;
        @supports not (padding-inline-end: 50px) {
            [dir=ltr] & {
                padding-right: 50px;
            }
            
            [dir=rtl] & {
                padding-left: 50px;
            }
        }
    "
`;

exports[`padding logical properties to be transformed padding-inline-start to be transformed to fallback 1`] = `
"
        padding-inline-start: 50px;
        @supports not (padding-inline-start: 50px) {
            [dir=ltr] & {
                padding-left: 50px;
            }
            
            [dir=rtl] & {
                padding-right: 50px;
            }
        }
    "
`;
